deploy/docker-compose/docker-compose-discounts-react-baseline.yml

```diff
@@ -0,0 +1,172 @@
+# Docker compose file which uses build context plus mounted volumes for easier local development
+# Note: storefront-fixed cannot use the build context because the .patch files are located in a parent directory of the Dockerfile
+version: '3'
+services:
+  agent:
+    image: "datadog/agent:7.29.0"
```

Comment:
Should we try to get the Agent version more up to date? I think we're at 7.31.1 now. This could also be a separate issue/PR so it addresses all of them at once.

Reply:
> Looks good! Left a comment about the Agent version, but I think that can/should be a separate PR that updates all of the docker compose files in bulk.
> 
> The other comments are to update the path for trace origins since we're moving to Instruqt.

Yep agreed. I'll make a separate issue for the agent upgrade